repl/lib/js/main.js

```diff
@@ -1,3 +1,6 @@
+import sanctuary from 'sanctuary';
+import ramdaFantasy from 'ramda-fantasy';
```

Comment:
how do we keep latest in the repo when RF or Sanctuary get updated? It occurs to me that is why we were relying on CDN